public/apps/configuration/panels/role-list.tsx

```diff
@@ -119,15 +120,14 @@ export function RoleList(props: AppDependencies) {
         setRoleData(processedData);
         setErrorFlag(false);
       } catch (e) {
-        console.log(e);
         setErrorFlag(true);
```

Comment:
I think this per-page setting of the error flag makes it possible to selectively view pages for users granted `restapi:admin/*` permissions. Currently, the logic to show `Security` on the main menu or not only takes into account users who have been assigned one of the `plugins.security.restapi.roles_enabled` roles.

Reply:
not sure what you meant by this one. Can you elaborate?

Reply:
i.e. a User with permission to restapi:admin/internaluser should be able to view the list of internal users in dashboards, but get "You do not have permission..." for other screens.

Currently, its all or none. You can either see "Security" in the main menu or you cannot.